src/templates/app/user/metatrader.jsx

```diff
@@ -494,8 +495,10 @@ const Metatrader = () => (
                     </form>
 
                     <div id='frm_cashier'>
-                        <div className='gr-row demo-only invisible'>
-                            <p className='gr-8 gr-push-2 gr-12-m gr-push-0-m gr-padding-30'>{it.L('This demo account comes with [_1] of virtual funds. Please [_2]contact our customer support team[_3] to replenish virtual funds if your account balance is empty.', '<span class="symbols mt-currency" />10,000.00', `<a href="${it.url_for('contact')}">`, '</a>')}</p>
```

Comment:
can we then remove the js code that handles `mt-currency`?

Reply:
Noted and addressed.